embassy-stm32/src/rtc/mod.rs

```diff
@@ -185,8 +214,11 @@ impl Rtc {
             None => freqs.rtc_hse.unwrap(),
         }
 
+        #[cfg(any(rcc_h7))]
+        return freqs.rtc.unwrap();
```

Comment:
This should not be added. If you're going to do this, just add `rtc_hse` to clocks for h7, and set it to none in rcc.